purchases/src/test/java/com/revenuecat/purchases/PurchasesFactoryTest.kt

```diff
@@ -26,13 +27,16 @@ class PurchasesFactoryTest {
 
     @Before
     fun setup() {
-        clearAllMocks()
-
         purchasesFactory = PurchasesFactory(apiKeyValidatorMock)
 
         every { apiKeyValidatorMock.validateAndLog("fakeApiKey", Store.PLAY_STORE) } just runs
     }
 
+    @After
+    fun tearDown() {
+        clearAllMocks()
+    }
+
```

Comment:
moved the `clearAllMocks()` call to an `@After` method since having it in `setup` potentially means we left some mocks from other tests.